lib/csv/parser.rb

```diff
@@ -335,20 +364,26 @@ def prepare_regexp
       @column_separator = @options[:column_separator].to_s.encode(@encoding)
       @row_separator =
         resolve_row_separator(@options[:row_separator]).encode(@encoding)
-      @quote_character = @options[:quote_character].to_s.encode(@encoding)
-      if @quote_character.length != 1
-        raise ArgumentError, ":quote_char has to be a single character String"
-      end
+
+
       @backslash_character = "\\".encode(@encoding)
+      if @options[:quote_character].nil?
+        @quote_character = nil
+      else
+        @quote_character = @options[:quote_character].to_s.encode(@encoding)
+        if @quote_character.length != 1
+          raise ArgumentError, ":quote_char has to be a single character String"
```

Comment:
"nil or a single ...".

Reply:
Fixed it to ":quote_char has to be nil or a single character String".
https://github.com/ruby/csv/pull/79/commits/72e8e2086c4e3ab48926d029c82a48ce56e25c32